### ISSUE TITLE
Question: Can I feed a tensor got from another graph to the facet get predict results?

### ISSUE THREAD

**Issue by zoezhou1999** [davidsandberg/facenet#1100]
I try to train a model whose loss function needs loss got from identity cross-entropy. I tried to convert tensor to numpy array and save image and predict features using facenet, but failed. I have used methods on the Internet about converting tensor to numpy array like .eval(session = sess). So I am wondering whether it is possible to feed the tensor got from another session to facenet model and get results. Thank you !
